pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -487,6 +487,54 @@ class StateVectorCudaManaged
         applyMatrix(gate_matrix.data(), wires, adjoint);
     }
 
+    /**
+     * @brief Collapse the state vector after having measured one of the qubit.
+     *
+     * Note: The branch parameter imposes the measurement result on the given
+     * wire.
+     *
+     * @param wire Wire to measure.
+     * @param branch Branch 0 or 1.
+     */
+    void collapse(const std::size_t wire, const bool branch) {
```

Comment:
`const` is redundant for std::size_t and bool types!

```suggestion
    void collapse(std::size_t wire, bool branch) {
```